docs/UserGuide.md

```diff
@@ -187,7 +185,7 @@ Adds the contacts specified by index as favourites
 Format: `addfav i/INDICES`
 - Adds the contacts at the specified `INDICES` as favourites. The indices refer to comma-separated index numbers (i.e. index, index, index) shown in the displayed person list. Each index **must be a positive integer** 1,2,3, ...
```

Comment:
Can you help change this to `contact list`

Reply:
Resolved